alacritty_terminal/src/tty/windows/switchable_backends.rs

```diff
@@ -0,0 +1,188 @@
+/// Types to wrap either Conpty or Winpty backends.
+///
+/// When the `winpty` feature is enabled, these will be used in the Pty
+/// struct instead of the Conpty types.
```

Comment:
I don't think it's important where these will be used, that's documented where they are used. What should be documented is what they do, not how they are used.

I'd propose something like this:

Unless the `winpty` feature is disabled, the PTY backend will automatically fall back to WinPTY when the newer ConPTY API is not supported, as long as the user hasn't explicitly opted into the WinPTY config option.

Might be a bit of a long sentence, but I think that should get better across what *this* module does, not what other modules might do with this module.